### ISSUE TITLE
Add feature to draw lines

### ISSUE THREAD

**Issue by floka94** [opensheetmusicdisplay/opensheetmusicdisplay#940]
OSMD does not support to add lines below or above the notes. See this example:

![image](https://user-images.githubusercontent.com/29139420/104808823-02befe80-57e9-11eb-865d-c6adf45250c0.png)

In MusicXML i can draw such lines with the following tag:
_Start of a line:_
```
      <direction placement="below">
        <direction-type>
          <bracket type="start" number="1" line-end="none" line-type="solid" default-y="-69.20"/>
          </direction-type>
        </direction>
```
_End of a line:_

```
      <direction placement="below">
        <direction-type>
          <bracket type="stop" number="1" line-end="none"/>
          </direction-type>
        </direction>
```

Is this feature planned, or still not possiple to add because vexflow does not support this?

**Comment by sschmidTU** [comment]
Vexflow does support this, there's already a method in OSMD to draw lines:
```js
osmd.Drawer.DrawOverlayLine(start: PointF2D, stop: PointF2D, musicPage: GraphicalMusicPage,
                           color: string = "#FF0000FF", lineWidth: number = 0.2)
```
![image](https://user-images.githubusercontent.com/33069673/104847086-53645380-58de-11eb-832b-9321d2470c15.png)
details here:
https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/issues/651#issuecomment-577800543

But i assume you want to have the lines drawn from XML via the `bracket` tag. That is currently not yet supported.

**Comment by floka94** [comment]
Yes you are right, i want to have the lines drawn from XML. But nice to know that there is already a method implemented to draw a simple line. Maybe i wil try to implement this feature.

**Comment by sschmidTU** [comment]
If we would do this in OSMD, which would be nice, we'd have to respect and afterwards update the Skyline or Bottomline. (SkyBottomLine)
